<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FriendsDialog</class>
 <widget class="QDialog" name="FriendsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Friends</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_friends">
      <attribute name="title">
       <string>Friends</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QListWidget" name="friendsList">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="btnSearch">
           <property name="text">
            <string>Search Users</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="add_friend_btn">
           <property name="text">
            <string>Add Friend</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_pending">
      <attribute name="title">
       <string>Pending Requests</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QListWidget" name="pendingList">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="accept_btn">
           <property name="text">
            <string>Accept</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="decline_btn">
           <property name="text">
            <string>Decline</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_sent">
      <attribute name="title">
       <string>Sent Requests</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QListWidget" name="sentList">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="cancel_btn">
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="refresh_btn">
     <property name="text">
      <string>ðŸ”„ ØªØ­Ø¯ÙŠØ«</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_close">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui> 